personal_profile/tests.py

```diff
@@ -31,13 +33,38 @@ def profile_1(db, user_1):
     return profile_1
 
 
+@pytest.mark.django_db
 class TestProfileModel:
-    # Testing different Personal Profile attributes
+    # Testing different Personal Profile saving and deletion
 
-    def test_newprofile(self, profile_1, user_1):
+    def test_profile_is_saved_correctly(self, profile_1, user_1):
         assert profile_1.company == 'Test Company'
         assert profile_1.user == user_1
         assert profile_1.about == 'Test About'
         assert profile_1.birth_date == datetime.date(1995, 12, 10)
         assert profile_1.profile_pic is not None
         assert profile_1.resume is not None
+
+    def test_profile_is_deleted(self, profile_1):
+        profile_1.save()
+        assert profile_1 in PersonalProfile.objects.all()
+        profile_1.delete()
+        assert profile_1 not in PersonalProfile.objects.all()
+
+    def check_name_of_files(self, profile_1, user_1):
+        assert profile_1.profile_pic == 'image_user_1.jpg'
+        assert profile_1.resume == 'resume_user_1.jpg'
+
+    def check_placement_of_files(self, profile_1, user_1):
```

Comment:
same here search for the profile and compare it with fixture content

Reply:
done